app/src/main/res/values/strings.xml

```diff
@@ -223,4 +224,16 @@
   <string name="add_profile_required">*Required</string>
   <string name="back_button">Back Button</string>
   <string name="next">Next</string>
+  <!-- OptionsActivity -->
+  <string name="key_story_text_size">key_story_text_size</string>
```

Comment:
Key should not come in `strings.xml` file rather they should be const vals in kotlin files because these keys are not needed to be translated.

Reply:
+1

Reply:
Done,